core/client/fs/src/main/java/alluxio/client/block/stream/LocalFilePacketReader.java

```diff
@@ -155,6 +156,9 @@ public void close() throws IOException {
       try {
         NettyRPC.call(NettyRPCContext.defaults().setChannel(mChannel).setTimeout(READ_TIMEOUT_MS),
             new ProtoMessage(request));
+      } catch (Exception e) {
+        mChannel.close();
```

Comment:
no. close() marks a bit inside channel and from then on, isOpen returns false (which makes it as an unhealthy resource).